org.mozilla.Firefox.json

```diff
@@ -12,8 +12,11 @@
         "--share=ipc",
         "--share=network",
         "--socket=pulseaudio",
+        "--socket=wayland",
```

Comment:
Sure, in principle! Not directly relevant for Endless at the moment, and does the release branch of Firefox have Wayland support anyway? As best I can tell ([a](https://bugzilla.mozilla.org/show_bug.cgi?id=635134), [b](https://glandium.org/blog/?p=3899)) only nightlies have Wayland support.